### ISSUE TITLE
h2 through h5 all look the same

### ISSUE THREAD

**Issue by samanthasunne** [hackshackers/hackshackers-hugo#127]
Not sure why this is. But h2 - h5 elements all look the same. For example: https://www.hackshackers.com/groups/la-plata/

**Comment by samanthasunne** [comment]
Also, fwiw, the headers look fine in the previews of Mardown pages, but then messed up in the actual published versions

**Comment by ludstuen90** [comment]
I took a look at the code, and it looks like the designers had made a conscious choice to funnel all of the sub-headers within posts to render as the same size on the site. It's a common size that appears in other parts of the site as well, eg. headers of the search results, sub-headers of page listings). 

We could definitely look at making this change, though I would classify it as a design change rather than a bug if we wanted to.

**Comment by samanthasunne** [comment]
Yes, let's change it - I'm retagging it as an enhancement. If it turns out to be a ton of work let me know
